contracts/PBAB+Collabs/GenArt721CoreV2WithEvents.sol

```diff
@@ -0,0 +1,793 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "../interfaces/0.8.x/IRandomizer.sol";
+import "../interfaces/0.8.x/IGenArt721CoreV2WithEvents.sol";
+
+import "@openzeppelin-4.5/contracts/utils/Strings.sol";
+
+import "@openzeppelin-4.5/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+
+pragma solidity 0.8.9;
+
+/**
+ * @title Powered by Art Blocks ERC-721 core contract.
+ * @author Art Blocks Inc.
+ */
+contract GenArt721CoreV2WithEvents is
+    ERC721Enumerable,
+    IGenArt721CoreV2WithEvents
+{
+    /// randomizer contract
+    IRandomizer public randomizerContract;
+
+    // generic platform event fields
+    bytes32 constant FIELD_ARTBLOCKS_OWNER = "owner";
```

Comment:
nit: I believe this can be a PBAB contract, so WDYT about naming this `FIELD_OWNER`?

Reply:
also, I have to change my earlier feedback on naming convention here...

I was thinking we would use this same event and the V3 convention of `owner` when implementing events on V3, but forgot that "Ownable" already implements an [OwnershipTransferred](https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable-OwnershipTransferred-address-address-) event, so keeping this as `"admin"` probably makes the most sense since the event won't be used in future versions.